### ISSUE TITLE
How can I add compound in trades

### ISSUE THREAD

**Issue by jasmeenPedalsup** [kernc/backtesting.py#1072]
Suppose I am running  backtest where it generate multiple trades in background. if I make profit in first trade then after in next trade my entry price will be initial price + profit and so on.

How can I achieve this?

**Comment by DomiTheProducer** [comment]
I would love to know this myself.

**Comment by singhay** [comment]
It already is the default, when opening a trade if you specify a percentage, then that's % is of overall equity at that time.

**Comment by kernc** [comment]
Indeed, [`Strategy.buy(size=.9999)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy) by default places orders for the total available equity (i.e. 99.99% of it) at any time.

Hope that's what was meant with:

> my entry price will be initial price + profit and so on.

If not, please clarify.
